ietf/templates/doc/opengraph.html

```diff
@@ -0,0 +1,16 @@
+{# Copyright The IETF Trust 2016-2020, All Rights Reserved #}
+{% load origin %}
+{% load static %}
+{% origin %}
+<meta property="og:title" content="{{ doc.title }} ({% if doc.get_state_slug == 'rfc' and not snapshot %}RFC {{ rfc_number }}{% else %}{{ name }}-{{ doc.rev }}{% endif %})">
+<meta property="og:url" content="{{ settings.IDTRACKER_BASE_URL }}{{ request.path }}">
+<meta property="og:image" content="{{ settings.IDTRACKER_BASE_URL }}{% static 'ietf/images/ietf-logo-nor-512.png' %}">
+<meta property="og:image:alt" content="Logo of the IETF">
```

Comment:
What about when it's _not_ an IETF draft? I think it's important to distinguish other streams as well as drafts not on a stream.